src/modules/components/transaction/transactionDataListItem/transactionDataListItemStructure/index.ts

```diff
@@ -0,0 +1,7 @@
+import { TransactionDataListItemStructure } from './transactionDataListItemStructure';
+
+export const TransactionDataListItem = {
+    Structure: TransactionDataListItemStructure,
+};
```

Comment:
That seems reasonable but this pattern matches the other DataListItem.Structures for the moment

Reply:
If the skeleton is in it's own folder, then the exports _have_ to be in the index of the parent because it won't make sense to be exporting the skeleton inside the index of the structure. The other DataListItem.Structure exports are incorrectly exported as well. Would you mind updating them so that they match the pattern set for the `ProposalDataListItem`?